learning-progression.md

```diff
@@ -132,54 +132,97 @@ basic.showIcon(IconNames.Butterfly)
 
 #### 2. Apply
 
+1. Write a program that shows a heart icon if the condition is true and a butterfly if the condition is false.  
+2. See what happens when you replace `true` or `false` with a mathematical relationship that will be evaluated as either true or false.
+For example, a relationship in which one number is greater than (`>`) or less than (`<`) another.  
+
+   Note that if you are trying to express equality, you will need two equals signs: `==`.
+
 #### 3. Present
 
+In the [Lab Notebook](README.md), include:
+1. A short narrative about the experiment.
+2. Short video of the operation of your code in 2.2.2.
 
+In the repository [programs folder](./programs), include:
+1. File `microbit-program-2-2-2.js` with the code you used in task 2.2.2.
 
 ### Step 6: Data types
 
+#### 1. Study
+
+To start thinking about the most basic type of data we can store, it is helpful to use the analogy of a light switch. When the switch is set to the higher power state, it is on, which can be interpretted to mean true or 1. When the switch is flipped to the lower power state, it is off, false, or 0. In this analogy, we can see that the same piece of data (the light switch being on or off) can be used to represent different kinds of information. 
+
+One of the most rudimentary types of data in javascript is a "Boolean," which has two possible values: `true` and `false`. We can apply this in the example that we used in the previous step to switch which block of code in the "`if`-`else`" statement is run.
+
+A variable is a symbol that is used to represent information. In a computer program, a variable can store information that is intended for use later on. The information stored in the variable can change as the program runs. For example, we may create a boolean with the value of `true`, and change it to `false` if a particular condition is met.
+
+In the previous step, we used the boolean literals `true` and `false` to change the outcome of our program. A boolean is also a [data type](https://makecode.microbit.org/javascript/types) for a variable. So, when a variable is declared, we can use the keyword `boolean` to indicate that it will store one of two possible boolean values.
+
 ```javascript
-let isHeart : boolean = true
+let myVar : boolean
+```
 
-if (isHeart) {
+Above is an example of how to declare a new variable in javascript. The keyword `let` indicates that the line that follows will be to declare a variable. The word after `let`, in this case `myVar`, will be interpretted as the name of this new variable. After the variable name, a colon must be placed before giving the new variable a type, in this case `boolean`.
+
+With this code, we have declared our variable and given it a type, however it has not been assigned any value. Since it has been declared as a `boolean`, its value can be either `true` or `false`. To assign this variable a value or change the value it currently holds, we would enter `myVar = true` or `myvar = false`.
```

Comment:
1. ..."have declared out variable of a specific type"...
2. "However ..." may read better as a separate sentence